bower.json

```diff
@@ -21,7 +21,8 @@
     "emojidata": "https://github.com/iamcal/emoji-data.git",
     "autosize": "~3.0.6",
     "webaudiorecorder": "https://github.com/higuma/web-audio-recorder-js.git",
-    "mp3lameencoder": "https://github.com/higuma/mp3-lame-encoder-js.git"
+    "mp3lameencoder": "https://github.com/higuma/mp3-lame-encoder-js.git",
+    "markdown-it": "^8.3.1"
```

Comment:
Good point. I don't usually use bower, I have my NPM configured to save exact by default, I didn't think that Bower wouldn't do that by default.